yoti_python_sdk/attribute.py

```diff
@@ -0,0 +1,18 @@
+class attribute:

```

Comment:
Shall we follow the same naming convention as `class Anchor` to start with uppercase for this `class attribute`?

Reply:
Yes, good spot